charts/agent/tests/security_context_test.yaml

```diff
@@ -0,0 +1,140 @@
+suite: Security Context tests
+templates:
+  - templates/daemonset.yaml
+kubernetesProvider:
+  scheme:
+    "v1/Node":
+      gvr:
+        version: "v1"
+        resource: "nodes"
+      namespaced: false
+  objects:
+  - apiVersion: v1
+    kind: Node
+    metadata:
+      name: fakenode
+    status:
+      nodeInfo:
+        osImage: fake-os-image
+tests:
+  - it: Ensure the securityContext for a privileged agent is the one defined
+    set:
+      privileged: true
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[?(@.name == "sysdig")].securityContext
+          value:
+            allowPrivilegeEscalation: true
+            privileged: true
+            readOnlyRootFilesystem: false
+            runAsNonRoot: false
+            runAsUser: 0
+
+  - it: Ensure the securityContext for a privileged agent is the one defined
+    set:
+      privileged: true
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[?(@.name == "sysdig")].securityContext
+          value:
+            allowPrivilegeEscalation: true
+            privileged: true
+            readOnlyRootFilesystem: false
+            runAsNonRoot: false
+            runAsUser: 0
```

Comment:
This seems to be a duplicate of the test before it.

Reply:
good catch, thanks! removed